packages/material-react-table/src/table/MRT_Table.tsx

```diff
@@ -93,6 +94,10 @@ export const MRT_Table = <TData extends Record<string, any>>({
     [columnPinning, enableColumnVirtualization, enableColumnPinning],
   );
 
+  const draggingColumnIndex = table
+    .getVisibleLeafColumns()
+    .findIndex((c) => c.id === draggingColumn?.id);
```

Comment:
Another scanning of all rows potentially on every render, even if no dragging is occurring or the dragging features are not enabled